Examples/UIExplorer/StatusBarExample.js

```diff
@@ -293,6 +305,21 @@ exports.examples = [{
   },
 }];
 
+if (android) {
```

Comment:
The way I see it, examples have to be written in a way that makes them as easy to understand as possible. The example right now has more than 70% platform specific code. Ideally, the iOS and android examples should not have that many discrepancies, so that the same example can be shared. If such a large amount of code cannot be shared, then it's better to just have them be separate.